tests/src/test/scala/cats/tests/ConstSuite.scala

```diff
@@ -66,6 +67,27 @@ class ConstSuite extends CatsSuite {
     }
   }
 
+  checkAll("Const[String, Int]", FunctorTests[Const[String, ?]].functor[Int, Int, Int])
+  checkAll("Functor[Const[String, ?]]", SerializableTests.serializable(Functor[Const[String, ?]]))
 
+  {
+    implicit val intCommutativeMonoid: CommutativeMonoid[Int] =
```

Comment:
Instead of defining this instance just for this test, I think that you should be able to use `CMono` from `Helpers.scala`. You can see an example of it being used in `FunctionSuite.scala`. Similarly for the commutative semigroup below (`CSemi`).

Reply:
sure, I will change it 👍